themes/startwords/assets/scss/article/_summary.scss

```diff
@@ -149,3 +149,33 @@ $preview-height: rem(250px);
         }
     }
 }
+
+
+.other-articles {
```

Comment:
probably want a comment at the top of this to explain, since it doesn't technically use the article summary template even though it is a summary of an article.

Reply:
it also would be a little more clear if the main heading/target for the rule was `.article-short-summary`.

Reply:
I wasn't sure how to do that! It seemed like it was too different from the other summary format. Would you revise the approach here, or just use the more readable target?